politeiawww/cmd/politeiawwwcli/config/config.go

```diff
@@ -54,6 +55,7 @@ type Config struct {
 	SkipVerify  bool   `long:"skipverify" description:"Skip verifying the server's certifcate chain and host name"`
 	Verbose     bool   `short:"v" long:"verbose" description:"Print verbose output"`
 	Silent      bool   `long:"silent" description:"Suppress all output"`
+	TestNet     bool   `long:"testnet" description:"use testnet params"`
```

Comment:
Would prefer that the testnet flag option be moved to the `NewInvoiceCmd` instead of being a global option.   politeiawwwcli doesn't differentiate testnet and mainnet so having a global testnet flag may cause confusion.

Reply:
ah good point